dom/src/main/scala-2/fs2/dom/Dom.scala

```diff
@@ -90,6 +90,23 @@ object Document {
 }
 
 trait HtmlDocument[F[_]] extends Document[F]
+object HtmlDocument {
+
+  implicit def ops[F[_]](element: HtmlDocument[F]): Ops[F] = new Ops(
+    element.asInstanceOf[dom.HTMLDocument]
+  )
+
+  private implicit def toJS[F[_]](element: HtmlDocument[F]): dom.HTMLDocument =
```

Comment:
This is `private` so we don't end up having clashing methods while working inside `fs2.dom`. I think it better matches the `extension` method semantics in scala 3 to be object private.